R/pkg/R/functions.R

```diff
@@ -818,6 +818,7 @@ setMethod("factorial",
 #' first(df$c, TRUE)
 #' }
 #' @note first(characterOrColumn) since 1.4.0
+#' @note the function is non-deterministic because its result depends on order of rows.
```

Comment:
I think `@note` is used for version specification in R but just use `Note:` or `Note that blabla`.

Reply:
yes, for now it is.
in this case, and a "Note:..." after L806  "value it sees when na.rm is set to true. If..."